package.json

```diff
@@ -404,13 +404,14 @@
   },
   "resolutions": {
     "underscore": "1.13.4",
-    "@types/slate": "0.47.2",
+    "@types/slate": "0.47.9",
     "@microsoft/api-extractor-model": "7.23.3",
     "@rushstack/node-core-library": "3.49.0",
     "@rushstack/rig-package": "0.3.13",
     "@rushstack/ts-command-line": "4.12.1",
     "@storybook/react/webpack": "5.74.0",
-    "node-fetch": "2.6.7"
+    "node-fetch": "2.6.7",
+    "slate-dev-environment@^0.2.2": "patch:slate-dev-environment@npm:0.2.5#.yarn/patches/slate-dev-environment-npm-0.2.5-9aeb7da7b5.patch"
```

Comment:
This is working for Grafana now, but should we incorporate that patch also/only to grafana-ui? I am not sure if there are dependents which do not those patches but would likely need them.